### PR TITLE
PAL-798 renamed top-level values to use kebab-case whenever referring…

### PR DESCRIPTION
**Pull request by dev930018**
… to chart names

### CODE REVIEW

File: charts/kafka/charts/zookeeper/templates/zookeeper-pv.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 {{- $namespace := include "palisade.namespace" . }}
 {{- $zookeepername := include "zookeeper.fullname" . }}
-{{- if .Values.global.persistence.statefulSet.pv.enabled }}
+{{- if .Values.global.persistence.statefulset.pv.enabled }}
```
Comment (kg981167224) [inline charts/kafka/charts/zookeeper/templates/zookeeper-pv.yaml]: stateful-set ????

File: charts/redis/templates/redis-pv-slave.yaml [hunk 2]
```diff
@@ -14,8 +14,8 @@
 
 {{- $namespace := include "palisade.namespace" . }}
 {{- $redisname := include "redis.fullname" . }}
-{{- if .Values.global.persistence.statefulSet.pv.enabled }}
-{{- $config := $.Values.global.persistence.redisSlave }}
+{{- if .Values.global.persistence.statefulset.pv.enabled }}
+{{- $config := (index $.Values.global.persistence "redis-slave") }}
```
Comment (kg981167224) [inline charts/redis/templates/redis-pv-slave.yaml:18]: Is there another ticket to change this to redis-worker?

File: values.yaml [hunk 3]
```diff
@@ -220,8 +223,8 @@ global:
         volumeHandle: null
         volumePath: "/"
 
-    # redisSlave: Readable store used by redis slave nodes for persistence and caching
-    redisSlave:
+    # redis-slave: Readable store used by redis slave nodes for persistence and caching
+    redis-slave:
       name: redis-persistence-slave
```
Comment (kg981167224) [inline values.yaml:228]: See previous comment about renaming

**Review by kg981167224 — CHANGES_REQUESTED** [review]

File: charts/redis/templates/redis-pv-slave.yaml (hunk 2 above)
Comment (dev930018) [reply charts/redis/templates/redis-pv-slave.yaml:18]: yup

File: values.yaml [hunk 4]
```diff
@@ -264,8 +267,8 @@ global:
         volumeHandle: null
         volumePath: "/"
 
-    # global.persistence.statefulSet.pv.enabled: Global variable that controls the generation of statefulSet pv/pvc(s)
-    statefulSet:
+    # global.persistence.statefulset.pv.enabled: Global variable that controls the generation of statefulset pv/pvc(s)
+    statefulset:
```
Comment (pd104923) [inline values.yaml]:
```suggestion
    # global.persistence.stateful-set.pv.enabled: Global variable that controls the generation of stateful-set pv/pvc(s)
    stateful-set:
```

**Review by pd104923 — CHANGES_REQUESTED** [review]
One cheeky change

**Review by pd104923 — APPROVED** [review]

**Review by dev-958 — APPROVED** [review]

**Pull request merged**